pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -147,24 +147,24 @@ type ShootStatus struct {
 type Addons struct {
 	// KubernetesDashboard holds configuration settings for the kubernetes dashboard addon.
 	// +optional
-	KubernetesDashboard *KubernetesDashboard `json:"kubernetes-dashboard,omitempty"`
+	KubernetesDashboard *KubernetesDashboard `json:"kubernetes-dashboard,omitempty" protobuf:"bytes,1,opt,name=kubernetesDashboard"`
```

Comment:
Are the names here correct (`kubernetes-dashboard` vs. `kubernetesDashboard`)?

Reply:
`-` is not  allowed in proto message field name. Ref https://developers.google.com/protocol-buffers/docs/reference/proto2-spec